assets/app/actions/siteActions.js

```diff
@@ -77,21 +77,35 @@ export default {
     }).catch(error => alertActions.alertError(error.message));
   },
 
+  createPR(site, head, base) {
+    return github.createPullRequest(site, head, base).then((pr) => {
+      return github.mergePullRequest(site, pr);
+    }).then(() => {
+      return github.deleteBranch(site, head);
+    }).then(() => {
+      this.fetchBranches(site);
+    }).then(() => {
+      return alertActions.alertSuccess(`${head} merged successfully`);
+    }).catch(error => alertActions.httpError(error.message));
```

Comment:
Sounds reasonable. If the network drops in the middle of deleting the branch, then there's not much of a recovery opportunity. We can wait and see if something actually is a problem before trying to recover "nicely"-- in this case, manual recovery isn't complicated and possible.